src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import cn from 'classnames';
+import { FilterType } from '../../types/FilterType';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todosFromServer: Todo[];
+  selectedFilter: FilterType;
+  setSelectedFilter: (filter: FilterType) => void;
+}
+
+export const Footer = ({
+  todosFromServer,
+  selectedFilter,
+  setSelectedFilter,
+}: Props) => {
+  const AmountOfTodosToComplete = todosFromServer.reduce(
+    (sum, todo) => sum + +!todo.completed,
+    0,
+  );
```

Comment:
Such casting of types is not very reliable, it is better to `filter` the array with all uncompleted todos and take its length